docs/source/index.rst

```diff
@@ -19,7 +19,7 @@ modularity and versatility for a broad set of industry use cases.
    whatsnew
    key_concepts
    getting_started
```

Comment:
The developing applications topic is important and we should leave it in the table of contents

Reply:
Agreed.  I had moved it into the Getting Started - Run Fabric Section, but it should stay as part of the main TOC. I got the same feedback from others as well, "Developing Applications" deserves its own section.  I have moved it back.